src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Contact {
+	
+	  public final String value;
+	  private boolean isPrivate;
+	   
+	  public Contact(String value,boolean isPrivate,String Reg,String problem) throws IllegalValueException{
```

Comment:
- variable name `Reg` should be starting with small letter (please check the coding guidelines!)
- please use meaningful names (probably full words) for variable names.